### ISSUE TITLE
ERROR! Unexpected Exception: name 'urllib2' is not defined

### ISSUE THREAD

**Issue by Adrenocortico** [LaunchKit/LaunchKit#13]
When I run `vagrant up --provision` this is what happens
```zsh
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/trusty64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: Setting the name of the VM: LaunchKit_default_1470218376948_6255
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 9100 (guest) => 9100 (host) (adapter 1)
    default: 9101 (guest) => 9101 (host) (adapter 1)
    default: 9102 (guest) => 9102 (host) (adapter 1)
    default: 9103 (guest) => 9103 (host) (adapter 1)
    default: 9104 (guest) => 9104 (host) (adapter 1)
    default: 9105 (guest) => 9105 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default:
    default: Guest Additions Version: 4.3.36
    default: VirtualBox Version: 5.0
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /vagrant => /Users/Francesco/LaunchKit
==> default: Running provisioner: ansible...
    default: Running ansible-playbook...
[DEPRECATION WARNING]: Instead of sudo/sudo_user, use become/become_user and
make sure become_method is 'sudo' (default).
This feature will be removed in a
future release. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
ERROR! Unexpected Exception: name 'urllib2' is not defined
to see the full traceback, use -vvv
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

I run it on **Mac OS 10.12 (16A254g)**
**vboxmanage --version : 5.0.26r108824**
**Vagrant 1.8.5**

**Comment by mplanchard** [comment]
I found this issue doing a Google search to resolve a similar issue. The problem in my case was that I had installed Python 3, which had overwritten the default `pip`, so ansible has been installed to the system Python 3, rather than Python 2.7. Since ansible doesn't run on Python 3 (although it's planned for a release later this year), it was failing with the `urllib2` error. The solution in my case was to uninstall the incorrectly installed ansible (`sudo pip uninstall ansible`) and install ansible specifically with `sudo pip2.7 install ansible`.
